src/engine/debug/debug.h

```diff
@@ -3,7 +3,7 @@
 
 #include "lumix.h"
 #include "core/iallocator.h"
-#include "core/mt/sync.h"
+#include "core/MT/sync.h"
```

Comment:
I can do it either way: rename MT and friends or change all includes. Decision is up to you.